translations/en.i18n.json

```diff
@@ -17,6 +17,8 @@
       "prev": "Prev",
       "startOver": "Start over",
       "pictureInPicture": "Picture in picture",
+      "pictureInPictureOpenToolTip": "Expand",
+      "pictureInPictureOpenAriaLabel": "Close picture in picture",
```

Comment:
is this correct? label key is open but value is meaning close - is this correct?

Reply:
I meant when PIP is open. I can add "is"

Reply:
so I think it is more correct to call it according to the text it represents and not the state in which it is active.
Let's run this by Ron and possibly a technical writer to come with better naming and strings as I think now we

Reply:
done